setup.py

```diff
@@ -16,11 +16,19 @@ def readme():
         'Development Status :: 3 - Alpha',
         'License :: OSI Approved :: MIT License',
         'Natural Language :: English',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.6',
         'Intended Audience :: Developers',
         ],
       py_modules = ['sqla_yaml_fixtures'],
-      install_requires = ['SQLAlchemy', 'PyYAML'],
+      install_requires = [
+          'SQLAlchemy',
+          'PyYAML'
+      ],
+      extras_require={
+          ':python_version=="2.7"': ['backports.functools_lru_cache'],
```

Comment:
Nice you used the modern way. But most people have a `pip` version that does not support it. I am ok to using this but a clear error message saying "upgrade pip" should be given for old versions of pip.